app/server/lib/sendAppPage.ts

```diff
@@ -121,15 +121,15 @@ export function makeMessagePage(staticDir: string) {
   };
 }
 
+export type SendAppPage = (req: express.Request, resp: express.Response, options: ISendAppPageOptions) => Promise<void>;
```

Comment:
Minor nitpick - Do you think it would be worth renaming this to `SendAppPageFunction` or `AppPageSender` or similar? I initially thought this would be an object representing a Page, rather than a function for sending the page to the client. I think using either `Function` in the name or making the name into a verb would clear that up.